android-components/components/support/utils/src/main/java/mozilla/components/support/utils/ColorUtils.kt

```diff
@@ -18,12 +19,12 @@ object ColorUtils {
     }
 
     /**
-     * Returns true if the color is dark enough that white text should be used on top of it.
+     * @return true if the given [color] is dark enough that white text should be used on top of it.
      */
     @JvmStatic
     @SuppressWarnings("MagicNumber")
     fun isDark(@ColorInt color: Int): Boolean {
-        if (color == Color.WHITE || Color.alpha(color) < 128) {
```

Comment:
This method was leading to unexpected results with various colours, notably using green. Updated to use `calculateLuminance` which does this calculations for us

Reply:
✨